src/custom/state/price/updater.ts

```diff
@@ -120,13 +124,22 @@ export default function FeesUpdater(): null {
     // Don't refetch if the amount is missing
     if (!amount) return
 
+    const unsupportedToken =
+      gpUnsupportedTokens?.[sellToken.toLowerCase()] || gpUnsupportedTokens?.[buyToken.toLowerCase()]
+
+    // IS an unsupported token and it's been greater than the threshold time
+    const unsupportedNeedsCheck =
+      unsupportedToken && Date.now() - unsupportedToken.dateAdded > UNSUPPORTED_TOKEN_REFETCH_CHECK_INTERVAL
```

Comment:
CAn you use function as it's done in lines 139 and 140. Makes it easier to read.

Reply:
done!